build_tools/third_party/flatcc/CMakeLists.txt

```diff
@@ -55,3 +54,22 @@ external_cc_library(
     "include/flatcc/reflection/flatbuffers_common_reader.h"
   PUBLIC
 )
+
+# Define our own binary target for the CLI, for host builds only.
+# flatcc's `flatcc_cli` target renames itself to `flatcc` or `flatcc_d`
+# depending on the build configuration, so we prefer to just set our own name.
+if(NOT CMAKE_CROSSCOMPILING)
```

Comment:
Why make this only for host builds?

Reply:
Mostly to match how the other CLIs are set up. Some tools may be possible to build for target platforms/architectures, but this consistently uses them from host builds. This can evolve over time, but flatcc was giving me enough trouble already that I just settled for what worked...